Makefile

```diff
@@ -304,10 +305,10 @@ $(MYGOBIN)/helmV2:
 	( \
 		set -e; \
 		d=$(shell mktemp -d); cd $$d; \
-		tgzFile=helm-v2.13.1-linux-amd64.tar.gz; \
+		tgzFile=helm-v2.13.1-$(UNAME)-amd64.tar.gz; \
 		wget https://storage.googleapis.com/kubernetes-helm/$$tgzFile; \
```

Comment:
Suggest replacing `wget` to `curl -Lo kustomize ...<URL>` since curl is more commonly used in macOS installation.